app/models/KeywordStatus.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+object KeywordStatus extends Enumeration {
+  type KeywordStatus = Value
+  val APPROVED, REJECTED, REPLACED = Value
+
+  def format(status: KeywordStatus): String =
+    status match {
+      case APPROVED => "Approved"
+      case REJECTED => "Rejected"
+      case REPLACED => "Renamed"
+    }
+}
```

Comment:
Is this model used anywhere?

Reply:
Newer version this model has now been created and is in use